### ISSUE TITLE
Add wrapper to Recipe Favorite icon

### ISSUE THREAD

**Issue by caitlynmayers** [caitlynmayers/dukes#138]
### Location
All recipes - http://test-dukes.pantheonsite.io/recipe/apple-cobbler/

### Description
The heart favorite icon is missing clear context to non-sighted users that indicates how to interact with that icon. Please add a wrapper to the icons with an aria label applied to give users the appropriate context for A conformance.

`aria-label="save recipe"`

**Comment by caitlynmayers** [comment]
Fix verified.
